Gopkg.toml

```diff
@@ -75,19 +75,19 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.14.0"
```

Comment:
Do we really need to upgrade to 1.14 given that it's just released?

Reply:
I guess why not? But if you have any concerns, I'm fine waiting a bit (maybe after 1.14.1 is out).

Reply:
Yes, let's wait for 1.14 to bake a bit more.

Reply:
Ok I'll make another PR containing just the added tests. Then we can merge this one later.